dbt_subprojects/dex/models/bot_trades/_schema.yml

```diff
@@ -0,0 +1,134 @@
+          name: token_sold_address
+          description: "Contract address of the token sold"
+        - &fee_percentage_fraction
+          name: fee_percentage_fraction
+          description: "The trading fee in percent, as a fraction of 100"
+        - &fee_usd
+          name: fee_usd
+          description: "USD value of the fee at time of execution"
+        - &fee_token_amount
+          name: fee_token_amount
+          description: "Value of the fee paid at time of execution in the original currency"
+        - &fee_token_symbol
+          name: fee_token_symbol
+          description: "Token symbol for fee token"
+        - &fee_token_address
+          name: fee_token_address
+          description: "Contract address of the fee token"
+        - &project
+          name: project
+          description: "Project name of the DEX"
+        - &version
+          name: version
+          description: "Version of the contract built and deployed by the DEX project"
+        - &token_pair
+          name: token_pair
+          description: "Token symbol pair for each token involved in the trade"
+        - &project_contract_address
+          name: project_contract_address
+          description: "Project contract address which executed the trade on the blockchain"
+        - &user
+          name: user
+          description: "Address which initiated the trade"
+        - &tx_hash
+          name: tx_hash
+          description: "Unique transaction hash value tied to each transaction on the DEX"
+        - &evt_index
+          name: evt_index
+          description: "Index of the corresponding trade event"
+        - &is_last_trade_in_transaction
+          name: is_last_trade_in_transaction
+          description: "Wether the trade is the last hop of the trade transaction, in case of a multi-hop trade"
+
+  - name: banana_gun_ethereum_bot_trades
+    meta:
+      blockchain: ethereum
+      sector: dex
+      project: banana_gun
+      contributors: whale_hunter
+    config:
+      tags: ["evm", "dex", "banana_gun", "trades"]
+    description: >
+      Banana Gun trades on Ethereum
+    data_tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - blockchain
+            - tx_hash
+            - evt_index
+      - check_bot_trades_seed:
+          seed_file: ref('banana_gun_ethereum_trades_seed')
+          blockchain: ethereum
+
+  - name: banana_gun_base_bot_trades
+    meta:
+      blockchain: base
+      sector: dex
+      project: banana_gun
+      contributors: whale_hunter
+    config:
+      tags: ["evm", "dex", "banana_gun", "trades"]
+    description: >
+      Banana Gun trades on Base
+    data_tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - blockchain
+            - tx_hash
+            - evt_index
+      - check_bot_trades_seed:
+          seed_file: ref('banana_gun_base_trades_seed')
+          blockchain: base
```

Comment:
no need to add the blockchain filter here, since the seed is only for this chain.